cirq/ops/pauli_string.py

```diff
@@ -41,10 +41,14 @@
 if TYPE_CHECKING:
     import cirq
 
+TDefault = TypeVar('TDefault')
+TKey = TypeVar('TKey', bound=raw_types.Qid)
+TKey2 = TypeVar('TKey2', bound=raw_types.Qid)
```

Comment:
Nit: Add some docstring to explain what these mean.
Question: Why TKey2? Is there going to be a Tkey3, ... TkeyN eventually as the key types grow?

Reply:
I split them into `TKeyOther` and `TKeyNew` for clarity.